javascript/javascript1/week3/travel-time.js

```diff
@@ -0,0 +1,35 @@
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+};
+
+const travelInformation1 = {
+  speed: 50,
+  destinationDistance: 38,
+};
+
+function formatTimeInString(hours, minutes) {
+  const hourText = hours === 1 ? 'hour' : 'hours';
+  const minuteText = minutes === 1 ? 'minute' : 'minutes';
+  let output = '';
+  if (hours !== 0 || minutes === 0) {
+    output += `${hours} ${hourText}`;
+  }
+  if (minutes > 0) {
+    if (hours > 0) output += ' and ';
+    output += `${minutes} ${minuteText}`;
+  }
+  return output;
```

Comment:
This was difficult to read, especially as one `if else` uses short hand and then another one is with curly brackets.
It's a good place to have a code comment, explaining what is going on.
Here is my approach:
```
if (hours && minutes === 0) {
    return `${hours} ${hourText}`
  } else if (hours && minutes > 0) {
      return `${hours} ${hourText} and ${minutes} ${minuteText}`
  } else {
      return `${minutes} ${minuteText}`
  }
```
Note that `hours !== 0` is the same as `hours`